beta1/integrated-build.json

```diff
@@ -93,8 +93,8 @@
         "strategy": {
           "type": "STI",
           "stiStrategy": {
-            "image": "openshift/ruby-20-centos",
-            "scripts": "https://raw.githubusercontent.com/openshift/ruby-20-centos/master/.sti/bin"
+            "image": "openshift/ruby-20-centos7",
+            "scripts": "https://raw.githubusercontent.com/openshift/sti-ruby/master/2.0/.sti/bin"
```

Comment:
The problem here is we are not maintaining ruby-20-centos anymore. It'll be there but we don't know how long, the current one that should be used is the one @jhadvig changed the docs to.